calicoctl/resourcemgr/tier.go

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2016 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package resourcemgr
+
+import (
+	"github.com/tigera/libcalico-go/lib/api"
+	"github.com/tigera/libcalico-go/lib/api/unversioned"
+	"github.com/tigera/libcalico-go/lib/client"
+)
+
+func init() {
+	registerResource(
+		api.NewTier(),
+		api.NewTierList(),
+		[]string{"NAME"},
+		[]string{"NAME", "ORDER"},
+		map[string]string{
```

Comment:
I don't see why we shouldn't have the order shown by default.  It's only two columns, and there is no reason the wide output can't be the same as the non-wide output for some resources.